src/BuildScriptGenerator/DotNetCore/DotNetCoreSdkVersions.cs

```diff
@@ -15,7 +15,7 @@ public static class DotNetCoreSdkVersions
         public const string DotNet50SdkVersion = "5.0.408";
         public const string DotNet60SdkVersion = "6.0.301";
         public const string DotNet60RtmSdkVersion = "6-0-100-rtm-21527-11";
-        public const string DotNet70SdkVersion = "7.0.100-preview.5.22307.18";
+        public const string DotNet70SdkVersion = "7.0.100-preview.6.22352.1";
```

Comment:
Can we add a test to check if versions are added to the storage account?

Reply:
Test added and tested locally:
![image](https://user-images.githubusercontent.com/86327553/179076937-4bb9a4d8-eab3-4c73-9b16-fe0eaf66cb0d.png)